src/components/Person/Person.jsx

```diff
@@ -2,0 +1,22 @@
+import React from 'react';
+
+export const Person = ({ person }) => {
+  // eslint-disable-next-line no-param-reassign
+  const isMarried = person.partnerName;
+  const partnerType = person.sex === 'm' ? 'wife' : 'husband';
```

Comment:
The logic for determining `partnerType` uses 'm' and 'f'. Ensure this is consistent with the `sex` values in `App.jsx`.